Update Makefile (extracting spark version section)

arundasan91 opened the pull request:
There was an additional `$` sign in `awk` portion of line 13 . Fixed it. Thanks @anfeng for pointing it out.

anfeng commented:
@arundasan91 I believe the extra $ is intentional. Otherwise, Makefile will replace with $5 by 5-th argument of Make command. Let's verify.

arundasan91 commented:
@anfeng , I just cloned COS and added the following lines to the Makefile with `awk '{print $$5}'`.

```
$(info SPARK_VERSION="$(SPARK_VERSION)")
$(info SPARK_HOME="$(SPARK_HOME)")
```

This was the output when I did `make clean`

```
SPARK_VERSION="1"
SPARK_HOME="/usr/local/spark"
```

I changed `awk '{print $$5}'` to `awk '{print $5}'` in Makefile and ran `make clean` again to get this:

```
SPARK_VERSION="   /___/ "
SPARK_HOME="/usr/local/spark"
```

So, I was wrong to start with :) Sorry. You are right, the extra $ is intentional. 

Thanks.
